apps/frontend/src/pages/access-management/libs/helpers/get-user-rows.helper.ts

```diff
@@ -2,7 +2,9 @@ import { type UserGetAllItemResponseDto } from "~/modules/users/users.js";
 
 import { type UserRow } from "../types/types.js";
 
-const getUserRows = (users: UserGetAllItemResponseDto[]): UserRow[] =>
+const getUserRows = (
+	users: Omit<UserGetAllItemResponseDto, "email">[],
```

Comment:
`Omit ` is a utility type that constructs a type by picking all properties from a given type and then removing a specific set of properties.
Do we need email on our tables of form?
We don't use it here, so we can remove it in this case, i think.
We get users without unnecessary information.

Reply:
I mean do we actually have users without emails here? or it's just a type without email. If it's the first option, then there shouldn't be an email property in this type if we don't return it. If the second option, then there is no point in omitting the email, because the data has it

Reply:
Added email to dtos and remove Omit